client/admin-pointers.js

```diff
@@ -37,11 +35,35 @@ jQuery( document ).ready( function( $ ) {
 			},
 		} );
 
-		target.pointer( options ).pointer( 'open' );
-		if ( pointer.dim ) {
-			target.css( 'z-index', 9999 );
-			$( 'body' ).append( '<div id="wcs-pointer-page-dimmer" class="wcs-pointer-page-dimmer"></div>' );
-			$( '#wcs-pointer-page-dimmer' ).fadeIn( 500 );
+		function open() {
+			const target = $( pointer.target );
```

Comment:
Why did this need to move?

If it were to return to the outer scope, would the later `$( pointer.target )` call be necessary?